data/json/terrain.json

```diff
@@ -757,6 +797,7 @@
         "type" : "terrain",
         "id" : "t_slime",
         "name": "slime",
+        "description": "A disgusting and slippery mess that could be used to stash things because who'd want to touch it?",
```

Comment:
Could use a description of the slime beyond "mess," and might recommend breaking this into two clauses.

Perhaps something like:

> A disgusting, slippery pile of slime. You suppose you could stash things in it, because who'd want to touch it?

Reply:
The suggestion could work, however I try to avoid re-using the name of the terrain/object repeatedly.